package.json

```diff
@@ -70,7 +70,7 @@
     "xo": "0.29.1"
   },
   "engines": {
-    "node": ">=14.17"
+    "node": "^17 || ^16 || ^14.17"
```

Comment:
lets instead keep the upper limit of this range open ended, like `^14.17 || >=16`. also, since this introduces a second lower bound, i think we also need to add `16.0.0` to the matrix verified in the GitHub Actions CI config

Reply:
Done and done.